### PR TITLE
applications: nrf5340_audio: Update the log for CI test system

### PR DESCRIPTION
**Pull request by rick1082**
Update the log for fitting the CI test system.
OCT-NONE
Signed-off-by: Jui-Chou Chung <jui-chou.chung@nordicsemi.no>

### CODE REVIEW

File: applications/nrf5340_audio/broadcast_source/main.c [hunk 1]
```diff
@@ -325,8 +325,8 @@ int main(void)
 
 	size_t ext_adv_size = 0;
 	size_t per_adv_size = 0;
-	const struct bt_data *ext_adv = NULL;
-	const struct bt_data *per_adv = NULL;
+	static const struct bt_data *ext_adv;
+	static const struct bt_data *per_adv;
```
Comment (koffes) [inline applications/nrf5340_audio/broadcast_source/main.c:329]: Suggest we take this out. This is another PR.

File: applications/nrf5340_audio/unicast_server/main.c [hunk 2]
```diff
@@ -482,10 +482,10 @@ void streamctrl_send(void const *const data, size_t size, uint8_t num_ch)
 int main(void)
 {
 	int ret;
+	static struct bt_data ext_adv_buf[CONFIG_EXT_ADV_BUF_MAX];
```
Comment (koffes) [inline applications/nrf5340_audio/unicast_server/main.c:485]: Same here.

**Review by koffes — CHANGES_REQUESTED** [review]

File: applications/nrf5340_audio/unicast_server/main.c (hunk 2 above)
Comment (rick1082) [reply applications/nrf5340_audio/unicast_server/main.c]: Created another PR for that:
https://github.com/nrfconnect/sdk-nrf/pull/14217

File: applications/nrf5340_audio/broadcast_source/main.c (hunk 1 above)
Comment (rick1082) [reply applications/nrf5340_audio/broadcast_source/main.c]: Created another PR for that:
https://github.com/nrfconnect/sdk-nrf/pull/14217

**Review by koffes — APPROVED** [review]

**Pull request merged**